Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN pip install --upgrade pip && \
     pip install --upgrade --extra-index-url https://pypi.anaconda.org/kbase/simple \
       -r requirements.txt \
       kbase-workspace-utils==0.0.13 && \
+      pandas && \
```

Comment:
lock this to at least a minor version, or a full version

Reply:
ive changed this to be a separate pip install outside of the pip install --upgrade but still in the same RUN statement